src/components/home-page/stats-section/index.tsx

```diff
@@ -1,37 +1,56 @@
 import StatItem from 'components/common/stat-item';
+let StatItems = [
+    {
+        bountyName: 'Bounties1',
+        bountyNumber: '135',
+        borderBottom: 'true',
+        borderRight: 'true',
+        removeBorderRightMedium: 'false',
+    },
+    {
+        bountyName: 'Bounties2',
+        bountyNumber: '135',
+        borderBottom: 'true',
+        borderRight: 'true',
+        removeBorderRightMedium: 'true',
+    },
+    {
+        bountyName: 'Bounties3',
+        bountyNumber: '1000',
+        borderBottom: 'true',
+        borderRight: 'true',
+        removeBorderRightMedium: 'false',
+    },
+    {
+        bountyName: 'Bounties4',
+        bountyNumber: '135',
+        borderBottom: 'false',
+        borderRight: 'false',
+        removeBorderRightMedium: 'false',
+    },
+];
 const StatsSection = () => (
     <section
         title="faq"
         className="grid sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-4"
     >
-        <StatItem
-            bountyName="Bounties1"
-            bountyNumber="135"
-            borderBottom="true"
-            borderRight="true"
-            removeBorderRightMedium="false"
-        />
-        <StatItem
-            bountyName="Bounties2"
-            bountyNumber="135"
-            borderBottom="true"
-            borderRight="true"
-            removeBorderRightMedium="true"
-        />
-        <StatItem
-            bountyName="Bounties3"
-            bountyNumber="1000"
-            borderBottom="true"
-            borderRight="true"
-            removeBorderRightMedium="false"
-        />
-        <StatItem
-            bountyName="Bounties4"
-            bountyNumber="135"
-            borderBottom="false"
-            borderRight="false"
-            removeBorderRightMedium="false"
-        />
+        {StatItems.map(
+            ({
+                bountyName,
+                bountyNumber,
+                borderBottom,
+                borderRight,
+                removeBorderRightMedium,
+            }) => (
+                <StatItem
```

Comment:
A `key` prop would be missing here.

Reply:
why? it is not `li`

Reply:
Basically, whenever you render a list of elements via `map`, you also need to add a unique identifier to each of them so that React assigns them a stable identity: https://reactjs.org/docs/lists-and-keys.html#keys. In this case, since `bountyName` is unique, something like this would be great:

```suggestion
                <StatItem key={bountyName} ... />
```

Reply:
Ohh
I thought that's only for each child in a list

Reply:
@hreyesm check it now

Reply:
LGTM